spec/integration/hanami/controller/head_spec.rb

```diff
@@ -64,11 +64,11 @@ def response
         expect(response.headers["Content-Language"]).to eq("en")
       end
 
-      it "doesn't send Content-Length header" do
+      it "does send Content-Length header" do
         get "/code/#{code}"
```

Comment:
Also, I did have to change this spec since we are no longer removing the body, the header is sent by `MockResponse`.

I was able to follow the code path:

1. Hanami does remove the header with `_empty_headers(res)`  https://github.com/hanami/controller/pull/268/files#diff-c3eb2b7cc639097f82ae3c2ed20be06bR714

2. `RackTest` at this line https://github.com/rack-test/rack-test/blob/master/lib/rack/mock_session.rb#L31
will return a `MockResponse` that had added the header. (My conclusion is the rack is adding the header again since there is a body)

WDYT? @jodosha